heft-plugins/heft-dev-cert-plugin/src/UntrustDevCertificatePlugin.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import TrustDevCertificatePlugin from './TrustDevCertificatePlugin';
+
+const PLUGIN_NAME: string = 'UntrustDevCertPlugin';
+
+export default class UntrustDevCertificatePlugin extends TrustDevCertificatePlugin {
+  public constructor() {
+    super(/* pluginName: */ PLUGIN_NAME, /* trustDevCert: */ false);
```

Comment:
The `trustDevCert: false` thing is kinda janky. That makes me assume that it just won't trust the cert, not that it'll untrust it.

Given that there are two fairly different code paths in `TrustDevCertificatePlugin`, I think just having some minor duplication between these two plugins (and not having one extend the other) is worth it.